src/main/java/ru/testtask/controller/MainController.java

```diff
@@ -91,7 +91,7 @@ public MainController(ProjectService projectService, ProjectDTOConverter project
         return new ResponseEntity<>(String.format("Project with ID %s removed successfully", id), HttpStatus.NO_CONTENT);
     }
 
-    @PreAuthorize("projectService.isCurrentUserOwnerOf(#id) or hasAuthority('MODERATOR')")
+    @PreAuthorize("hasAuthority('MODERATOR') ")
```

Comment:
What is the reason to change the behavior of the permission functionality

Reply:
Was testing one feature, rolled it back